lib/benchee.ex

```diff
@@ -87,6 +98,19 @@ for {module, moduledoc} <- [{Benchee, elixir_doc}, {:benchee, erlang_doc}] do
 
     defp is_formatter_module?(_), do: false
 
+    defp call_formatter_function(function, suite) do
+      if arity(function) == 1 do
+        function.(suite)
+      else
+        # This is to make our `output/2` functions work albeit it is preferred for the user
+        # to do something akin to: `fn suite -> Formatter.output(suite, my_opts_here) end`
+        # so that it ends up in the upper part of this if
+        function.(suite, @default_opts)
+      end
```

Comment:
This will be better with:
```elixir
defp call_formatter_function(function, suite) when is_function(function, 1) do
  function.(suite)
end

defp call_formatter_function(function, suite) when is_function(function, 2) do
  function.(suite, @default_opts)
end
```

Reply:
Yay for `is_function` - you should review my PRs more often there seems to be a lot of room for me to learn :grin: